src/main/java/org/zalando/nakadi/exceptions/IllegalScopeException.java

```diff
@@ -0,0 +1,18 @@
+package org.zalando.nakadi.exceptions;
+
+import java.util.Set;
+
+public class IllegalScopeException extends RuntimeException {
```

Comment:
@rcillo I made it just to feel the taste of that and for me it looks very nice. I am not sure that this point in the google doc is about it :)